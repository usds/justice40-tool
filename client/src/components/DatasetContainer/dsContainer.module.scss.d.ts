declare namespace DatasetContainerScssNamespace {
    export interface IDatasetContainerScss {
      datasetCardsContainer: string;
      returnToTop: string;
    }
  }

declare const DatasetContainerScssModule: DatasetContainerScssNamespace.IDatasetContainerScss & {
    /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
    locals: DatasetContainerScssNamespace.IDatasetContainerScss;
  };

  export = DatasetContainerScssModule;
