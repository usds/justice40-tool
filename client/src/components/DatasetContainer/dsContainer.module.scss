@use '../../styles/design-system.scss' as *;

.datasetCardsContainer {
  @include u-margin-top(4);
  display: flex;
  justify-content: space-between;
  flex-wrap: wrap;
}

.returnToTop {
  display: flex;
  justify-content: flex-end;
  @include u-margin-bottom(4);
}