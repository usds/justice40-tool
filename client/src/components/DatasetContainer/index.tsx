import React from 'react';
import {Link, useIntl} from 'gatsby-plugin-intl';
import {Grid} from '@trussworks/react-uswds';

import DatasetCard from '../DatasetCard';
import J40MainGridContainer from '../J40MainGridContainer';
import {hyphenizeString} from '../../../cypress/integration/common/helpers';

import * as styles from './dsContainer.module.scss';
import * as METHODOLOGY_COPY from '../../data/copy/methodology';
import {PAGES_ENDPOINTS} from '../../data/constants';
import DatasetsButton from '../DatasetsButton';


const DatasetContainer = () => {
  const intl = useIntl();

  return (
    <>
      <J40MainGridContainer fullWidth={true} blueBackground={false}>
        <J40MainGridContainer
          dataCy={`${hyphenizeString(METHODOLOGY_COPY.DATASETS.HEADING.defaultMessage)}-block`}>

          <Grid row>
            <Grid col={12}>
              <h2>{METHODOLOGY_COPY.DATASETS_RICH_TEXT.HEADING}</h2>
            </Grid>
          </Grid>

          <Grid row gap className={styles.alignItemsCenter}>
            <Grid desktop={{col: 7}}>
              <p>{intl.formatMessage(METHODOLOGY_COPY.DATASETS.INFO)}</p>
              <p>{intl.formatMessage(METHODOLOGY_COPY.DATASETS.CENSUS_DISCLAIMER)}</p>
            </Grid>
            <Grid desktop={{col: 1}}>
            </Grid>
            <Grid desktop={{col: 4}}>
              <DatasetsButton href= {'https://www.surveymonkey.com/r/6G9TQJ8'}/>
            </Grid>
          </Grid>

          <Grid row>
            <Grid col={12}>
              <div className={styles.datasetCardsContainer}>
                {METHODOLOGY_COPY.INDICATORS.map((card, index) => <DatasetCard
                  key={index}
                  datasetCardProps={card}
                />)}
              </div>
            </Grid>
          </Grid>

          <div className={styles.returnToTop}>
            <Link className={'usa-link'} to={PAGES_ENDPOINTS.METHODOLOGY}>
              {METHODOLOGY_COPY.RETURN_TO_TOP.LINK}
            </Link>
          </div>

        </J40MainGridContainer>
      </J40MainGridContainer>

    </>
  );
};

export default DatasetContainer;
