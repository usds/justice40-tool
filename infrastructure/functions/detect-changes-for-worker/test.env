REGION=us-east-1
STAGE=sit

# ESC Cluster name that will from the containers
ECS_CLUSTER=j40-sit-justice40-data-harvester-ECSCluster-ktXGGU9zjwkb

# VPC Private Subnet that has a NAT GAteway
VPC_SUBNET_ID=subnet-07e68cb57322f7b1f

# Names of the container and task names.
GDAL_TASK_DEFINITION=sit-justice40-data-harvester-gdal
GDAL_CONTAINER_DEFINITION=sit-justice40-data-harvester-osgeo-gdal
TIPPECANOE_TASK_DEFINITION=sit-justice40-data-harvester-tippecanoe
TIPPECANOE_CONTAINER_DEFINITION=sit-justice40-data-harvester-mgiddens-tippecanoe

# AWS Credentials
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
AWS_DEFAULT_REGION=us-east-1