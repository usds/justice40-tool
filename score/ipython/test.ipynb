{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "1a4c0c68",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "70b3a793",
   "metadata": {},
   "outputs": [],
   "source": [
    "df = pd.read_csv('data/fips_states.csv')  "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "c514aad8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>fips</th>\n",
       "      <th>state_name</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>Alabama</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2</td>\n",
       "      <td>Alaska</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>4</td>\n",
       "      <td>Arizona</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5</td>\n",
       "      <td>Arkansas</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>6</td>\n",
       "      <td>California</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   fips  state_name\n",
       "0     1    Alabama \n",
       "1     2     Alaska \n",
       "2     4    Arizona \n",
       "3     5   Arkansas \n",
       "4     6  California"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9ee44d9",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
