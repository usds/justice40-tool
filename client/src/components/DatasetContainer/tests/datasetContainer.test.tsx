import * as React from 'react';
import {render} from '@testing-library/react';
import {LocalizedComponent} from '../../../test/testHelpers';
import DatasetContainer from '../../DatasetContainer';

describe('rendering of the DatasetContainer', () => {
  const {asFragment} = render(
      <LocalizedComponent>
        <DatasetContainer />
      </LocalizedComponent>,
  );

  it('checks if various text fields are visible', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
