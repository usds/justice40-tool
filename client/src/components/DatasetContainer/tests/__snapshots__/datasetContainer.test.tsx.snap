// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering of the DatasetContainer checks if various text fields are visible 1`] = `
<DocumentFragment>
  <div
    class=""
  >
    <div
      class="grid-container-desktop-lg"
      data-cy="datasets-used-in-methodology-block"
      data-testid="gridContainer"
    >
      <div
        class="grid-row"
        data-testid="grid"
      >
        <div
          class="grid-col-12"
          data-testid="grid"
        >
          <h2>
            Datasets used in methodology
          </h2>
        </div>
      </div>
      <div
        class="grid-row"
        data-testid="grid"
      >
        <div
          class="grid-col-12"
          data-testid="grid"
        >
          <div>
            <div
              id="low-income"
            >
              <h3>
                Low income
              </h3>
              <div>
                
        Percent of a census tract's population in households where household income is at or below
        200% of the federal poverty level.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.census.gov/programs-surveys/acs"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Census's American Community Survey
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2015-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  All methodologies except for training and workforce development
                </li>
              </ul>
            </div>
            <div
              id="exp-agr-loss-rate"
            >
              <h3>
                Expected agriculture loss rate
              </h3>
              <div>
                
        Percent of agriculture value at risk from losses due to natural hazards. Calculated by dividing 
        the agriculture value at risk in a census tract by the total agriculture value in that census 
        tract. Fourteen natural hazards that have some link to climate change include: avalanche, 
        coastal flooding, cold wave, drought, hail, heat wave, hurricane, ice storm, landslide, 
        riverine flooding, strong wind, tornado, wildfire, and winter weather.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://hazards.fema.gov/nri/expected-annual-loss"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Federal Emergency Management Agency (FEMA)
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2014-2021
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Climate change methodology
                </li>
              </ul>
            </div>
            <div
              id="exp-bld-loss-rate"
            >
              <h3>
                Expected building loss rate
              </h3>
              <div>
                
        Percent of building value at risk from losses due to natural hazards. Calculated by dividing the 
        building value at risk in a census tract by the total building value in that census tract. 
        Fourteen natural hazards that have some link to climate change include: avalanche, 
        coastal flooding, cold wave, drought, hail, heat wave, hurricane, ice storm, landslide, 
        riverine flooding, strong wind, tornado, wildfire, and winter weather.
        
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://hazards.fema.gov/nri/expected-annual-loss"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Federal Emergency Management Agency (FEMA)
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2014-2021
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Climate change methodology
                </li>
              </ul>
            </div>
            <div
              id="exp-pop-loss-rate"
            >
              <h3>
                Expected population loss rate
              </h3>
              <div>
                
        Rate relative to the population in fatalities and injuries due to natural hazards each year. 
        Fourteen natural hazards that have some link to climate change include: avalanche, coastal 
        flooding, cold wave, drought, hail, heat wave, hurricane, ice storm, landslide, riverine 
        flooding, strong wind, tornado, wildfire, and winter weather.
        Population loss is defined as the Spatial Hazard Events and Losses or National Centers 
        for Environmental Information’s (NCEI) reported number of fatalities and injuries caused by the 
        hazard occurrence. To combine fatalities and injuries for the computation of population loss value, 
        an injury is counted as one-tenth (1/10) of a fatality. The NCEI Storm Events Database 
        classifies injuries and fatalities as direct or indirect. Both direct and indirect injuries 
        and fatalities are counted as population loss. This total number of injuries and fatalities 
        is then divided by the population in the census tract to get a per-capita rate of population risk.   
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://hazards.fema.gov/nri/expected-annual-loss"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Federal Emergency Management Agency (FEMA)
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2014-2021
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Climate change methodology
                </li>
              </ul>
            </div>
            <div
              id="energy-burden"
            >
              <h3>
                Energy cost burden
              </h3>
              <div>
                Average annual energy cost ($) divided by household income.
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.energy.gov/eere/slsc/low-income-energy-affordability-data-lead-tool"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Department of Energy (DOE) LEAD Score
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2018
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Clean energy and energy efficiency methodology
                </li>
              </ul>
            </div>
            <div
              id="pm-25"
            >
              <h3>
                PM2.5 in the air
              </h3>
              <div>
                
        Fine inhalable particles, with diameters that are generally 2.5 micrometers and smaller.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.epa.gov/ejscreen/technical-documentation-ejscreen"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Environmental Protection Agency (EPA) Office of Air and Radiation (OAR) fusion of model and monitor 
          data as compiled by EPA's EJSCREEN, sourced from EPA National Air Toxics Assessment (NATA), 2017 
          U.S. Department of Transportation (DOT) traffic data
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2017
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Clean energy and energy efficiency methodology
                </li>
              </ul>
            </div>
            <div
              id="diesel-pm"
            >
              <h3>
                Diesel particulate matter exposure
              </h3>
              <div>
                
        Mixture of particles that is part of diesel exhaust in the air.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.epa.gov/ejscreen/technical-documentation-ejscreen"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Environmental Protection Agency (EPA) National Air Toxics Assessment (NATA)
          as compiled by EPA's EJSCREEN        
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2014
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Clean transportation methodology
                </li>
              </ul>
            </div>
            <div
              id="traffic-vol"
            >
              <h3>
                Traffic proximity and volume
              </h3>
              <div>
                
        Count of vehicles (average annual daily traffic) at major roads
        within 500 meters, divided by distance in meters (not km).
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.epa.gov/ejscreen/technical-documentation-ejscreen"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Department of Transportation (DOT) traffic data as compiled by EPA's EJSCREEN      
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2017
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Clean transportation methodology
                </li>
              </ul>
            </div>
            <div
              id="house-burden"
            >
              <h3>
                Housing cost burden
              </h3>
              <div>
                
        The percent of households in a census tract that are both earning less than 80% of HUD Area Median 
        Family Income by county and are paying greater than 30% of their income to housing costs.    
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.huduser.gov/portal/datasets/cp.html"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Department of Housing & Urban Development’s
          (HUD) Comprehensive Housing Affordability Strategy dataset
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2014-2018
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Affordable and sustainable housing methodology
                </li>
              </ul>
            </div>
            <div
              id="lead-paint"
            >
              <h3>
                Lead paint
              </h3>
              <div>
                
        Percent of housing units built pre-1960, used as an indicator of potential lead paint exposure in 
        tracts with median home values less than 90th percentile
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.census.gov/programs-surveys/acs"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Census's American Community Survey
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2015-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Affordable and sustainable housing methodology
                </li>
              </ul>
            </div>
            <div
              id="median-home"
            >
              <h3>
                Low median home value
              </h3>
              <div>
                
        Median home value of owner-occupied housing units in the census tract.
       
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.census.gov/programs-surveys/acs"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Census's American Community Survey
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2015-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Affordable and sustainable housing methodology
                </li>
              </ul>
            </div>
            <div
              id="prox-haz"
            >
              <h3>
                Proximity to hazardous waste facilities
              </h3>
              <div>
                
        Count of hazardous waste facilities (Treatment, Storage, and Disposal Facilities and Large
        Quantity Generators) within 5 km (or nearest beyond 5 km), each divided by distance in kilometers.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://enviro.epa.gov/facts/rcrainfo/search.html"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Environmental Protection Agency (EPA) Treatment Storage, and Disposal Facilities
          (TSDF) data calculated from EPA RCRA info database as compiled by EPA’s EJSCREEN
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2020
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Reduction and remediation of legacy pollution methodology
                </li>
              </ul>
            </div>
            <div
              id="prox-npl"
            >
              <h3>
                Proximity to National Priorities List (NPL) sites
              </h3>
              <div>
                
        Count of proposed or listed NPL - also known as superfund - sites within 5 km (or nearest one
        beyond 5 km), each divided by distance in kilometers.      
        
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://enviro.epa.gov/facts/rcrainfo/search.html"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Environmental Protection Agency (EPA) CERCLIS database as compiled by EPA’s EJSCREEN        
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2020
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Reduction and remediation of legacy pollution methodology
                </li>
              </ul>
            </div>
            <div
              id="prox-rmp"
            >
              <h3>
                Proximity to Risk Management Plan (RMP) facilities
              </h3>
              <div>
                
        Count of RMP (potential chemical accident management plan) facilities within 5 km (or nearest
        one beyond 5 km), each divided by distance in kilometers.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.epa.gov/ejscreen/technical-documentation-ejscreen"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Environmental Protection Agency (EPA) RMP database as compiled by EPA’s EJSCREEN        
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2020
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Reduction and remediation of legacy pollution methodology
                </li>
              </ul>
            </div>
            <div
              id="waste-water"
            >
              <h3>
                Wastewater discharge
              </h3>
              <div>
                
        Risk-Screening Environmental Indicators (RSEI) modeled Toxic Concentrations at 
        stream segments within 500 meters, divided by distance in kilometers (km).
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.epa.gov/ejscreen/technical-documentation-ejscreen"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Environmental Protection Agency (EPA) Risk-Screening
          Environmental Indicators (RSEI) Model as compiled by EPA's EJSCREEN        
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2020
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Critical clean water and waste infrastructure
                </li>
              </ul>
            </div>
            <div
              id="asthma"
            >
              <h3>
                Asthma
              </h3>
              <div>
                
        Weighted percent of people who answer “yes” to both of the following questions: “Have you ever 
        been told by a doctor, nurse, or other health professional that you have asthma?” and the question
        “Do you still have asthma?”
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.cdc.gov/places/index.html"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Centers for Disease Control and Prevention (CDC) PLACES        
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2016-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Health burdens methodology
                </li>
              </ul>
            </div>
            <div
              id="diabetes"
            >
              <h3>
                Diabetes
              </h3>
              <div>
                
        Weighted percent of people ages 18 years and older who report having ever been
        told by a doctor, nurse, or other health professionals that they have
        diabetes other than diabetes during pregnancy.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.cdc.gov/places/index.html"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Centers for Disease Control and Prevention (CDC) PLACES        
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2016-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Health burdens methodology
                </li>
              </ul>
            </div>
            <div
              id="heart-disease"
            >
              <h3>
                Heart disease
              </h3>
              <div>
                
        Weighted percent of people ages 18 years and older who report ever having been told
        by a doctor, nurse, or other health professionals that they had angina or
        coronary heart disease.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.cdc.gov/places/index.html"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          Centers for Disease Control and Prevention (CDC) PLACES        
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2016-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Health burdens methodology
                </li>
              </ul>
            </div>
            <div
              id="life-exp"
            >
              <h3>
                Low life expectancy
              </h3>
              <div>
                
        Average number of years of life a person who has attained a given age can expect to live.
        
                <p>
                  <strong>
                    Note:
                  </strong>
                  
          Unlike most of the other datasets, high values of this indicator indicate low burdens. 
          For percentile calculations, the percentile is calculated in reverse order, so that the tract with 
          the highest median income relative to area median income (lowest burden on this measure) is at the 
          0th percentile, and the tract with the lowest median income relative to area median income 
          (highest burden on this measure) is at the 100th percentile.
        
                </p>
                
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.cdc.gov/nchs/nvss/usaleep/usaleep.html#data"
                    rel="noreferrer"
                    target="_blank"
                  >
                    
          CDC’s U.S. Small-area Life Expectancy Estimates Project (USALEEP)        
        
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2010-2015
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Health burdens methodology
                </li>
              </ul>
            </div>
            <div
              id="low-med-inc"
            >
              <h3>
                Low median Income
              </h3>
              <div>
                
        Median income of the census tract calculated as a percent of the area’s median income.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.census.gov/programs-surveys/acs"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Census's American Community Survey
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2015-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Training and workforce development methodology
                </li>
              </ul>
            </div>
            <div
              id="ling-iso"
            >
              <h3>
                Linguistic Isolation
              </h3>
              <div>
                
        The percent of limited speaking households, which are households where no one over age 14 speaks English well.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.census.gov/programs-surveys/acs"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Census's American Community Survey
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2015-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Training and workforce development methodology
                </li>
              </ul>
            </div>
            <div
              id="unemploy"
            >
              <h3>
                Unemployment
              </h3>
              <div>
                
      Number of unemployed people as a percentage of the civilian labor force
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.census.gov/programs-surveys/acs"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Census's American Community Survey
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2015-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Training and workforce development methodology
                </li>
              </ul>
            </div>
            <div
              id="poverty"
            >
              <h3>
                Poverty
              </h3>
              <div>
                
        Percent of a tract's population in households where the household income is at or below 100% of 
        the federal poverty level.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.census.gov/programs-surveys/acs"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Census's American Community Survey
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2015-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Training and workforce development methodology
                </li>
              </ul>
            </div>
            <div
              id="high-school"
            >
              <h3>
                High school degree achievement rate
              </h3>
              <div>
                
        Percent (not percentile) of people ages 25 years or older in a census tract whose
        education level is less than a high school diploma.
      
              </div>
              <ul>
                <li>
                  <span>
                    Responsible Party: 
                  </span>
                  <a
                    href="https://www.census.gov/programs-surveys/acs"
                    rel="noreferrer"
                    target="_blank"
                  >
                    Census's American Community Survey
                  </a>
                </li>
                <li>
                  <span>
                    Date range: 
                  </span>
                  2015-2019
                </li>
                <li>
                  <span>
                    Used in: 
                  </span>
                  Training and workforce development methodology
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
      <div>
        <a
          href="/en/methodology"
        >
          Return to top
        </a>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
